src/main/res/values/strings.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
 
-  <string name="action_failed">Action failed</string>
+  <string name="action_failed">Could not carry out action</string>
```

Comment:
This string is probably too long to be displayed in dialog title.

Reply:
What sort of action is it?

Reply:
It's a generic string for all failed actions. This dialog also has description with explanation, and it's not always something which can be retried later.